presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -90,6 +116,13 @@ protected ResultSet getTables(Connection connection, String schemaName, String t
     @Override
     public Optional<ColumnMapping> toPrestoType(ConnectorSession session, JdbcTypeHandle typeHandle)
     {
+        if (typeHandle.getJdbcType() == Types.OTHER) {
```

Comment:
I don't think we need to check the JDBC type here. The switch on the type name should be sufficient.